docs/overview/architecture.md

```diff
@@ -15,17 +16,61 @@ Vald uses a cloud-native architecture focusing on [Kubernetes](https://kubernete
 Some components in Vald use Kubernetes API to control the behavior of distributed vector indexes.
 Before reading this document, you need to have some understanding of the basic idea of cloud-native architecture and Kubernetes.
 
-The below image is Vald's architecture.
+### Concept
 
-<img src="../../design/Vald Future Architecture Overview.svg" />
+Here are the concepts of Vald.
+
+- Microservice based
+
+  Vald is designed based on the microservice architecture. Vald components are highly decoupled into small components and connected, which increases the overall agility and maintainability of Vald.
+
+- Containerized
+
+  All components in Vald are containerized, which means you can easily deploy Vald components on any environment.
+
+- Observability & Real-time monitoring
+
+  Vald supports pprof and Prometheus to monitor and trace the log in every Vald components.
```

Comment:
![badge](https://img.shields.io/badge/review-imo-orange.svg?style=flat-square)
Not only pprof and Prometheus exporter are the observability features. Please revise it like following:
```suggestion
  Every Vald components supports Cloud-Native based observability features, e.g. Prometheus exporter and Jaeger exporter.
```

Reply:
thank you for your comment. I think it is better 🤔
```suggestion
  All Vald components support Cloud-Native based observability features such as Prometheus and Jaeger.
```

Reply:
sounds good. :+1:
I prefer "Prometheus and Jaeger **exporter**".

Reply:
```suggestion
  All Vald components support Cloud-Native based observability features such as Prometheus and Jaeger exporter.
```